wsd/COOLWSD.cpp

```diff
@@ -1514,6 +1514,8 @@ void COOLWSD::innerInitialize(Poco::Util::Application& self)
     }
     UnitWSD::get().setWSD(this);
 
+    net::Defaults.maxExtConnections = Util::getMaxConcurrentTCPConnections(std::max<size_t>(3, MAX_CONNECTIONS));
```

Comment:
3 ?

Reply:
the used magic in same compilation unit for the very minimum value of this session limit threshold - but it gets validated at a later stage when used.